alacritty/src/cli.rs

```diff
@@ -280,44 +280,3 @@ impl Options {
         config
     }
 }
-
-#[cfg(test)]
-mod test {
-    use alacritty_terminal::config::{Config, DEFAULT_ALACRITTY_CONFIG};
-
-    use crate::cli::Options;
-
-    #[test]
-    fn dynamic_title_ignoring_options_by_default() {
-        let config: Config =
-            ::serde_yaml::from_str(DEFAULT_ALACRITTY_CONFIG).expect("deserialize config");
-        let old_dynamic_title = config.dynamic_title();
-
-        let config = Options::default().into_config(config);
-
-        assert_eq!(old_dynamic_title, config.dynamic_title());
-    }
-
-    #[test]
-    fn dynamic_title_overridden_by_options() {
-        let config: Config =
-            ::serde_yaml::from_str(DEFAULT_ALACRITTY_CONFIG).expect("deserialize config");
-
-        let mut options = Options::default();
-        options.title = Some("foo".to_owned());
-        let config = options.into_config(config);
-
-        assert!(!config.dynamic_title());
-    }
-
-    #[test]
-    fn dynamic_title_overridden_by_config() {
-        let mut config: Config =
-            ::serde_yaml::from_str(DEFAULT_ALACRITTY_CONFIG).expect("deserialize config");
-
-        config.window.title = Some("foo".to_owned());
-        let config = Options::default().into_config(config);
-
-        assert!(!config.dynamic_title());
-    }
```

Comment:
Why are you removing these tests instead of updating them?

Reply:
Oops, Will update after evlp2 merge